website/src/constants/index.tsx

```diff
@@ -19,6 +19,10 @@ export enum EXTERNAL_LINKS {
   SUBSCRIBE_NEWSLETTER='https://lists.cncf.io/g/cncf-openebs-announcements',
 };
 
+export const API = {
+  GITHUB_CONTRIBUTORS: 'https://api.github.com/repos/openebs/openebs-docs/contributors?q=contributions&order=desc'
```

Comment:
Should we take the github username and reponame from constants?

Reply:
it's already in `constants` file :)